pss/pss.go

```diff
@@ -27,12 +27,21 @@ import (
 // Pss is the top-level struct, which takes care of message sending
 type Pss struct {
 	localStore chunk.Store
+	tag        *chunk.Tag
+}
+
+// State exposes underling chunk states
+type State struct {
+	stateStored chunk.State // chunk stored locally
+	stateSent   chunk.State // chunk sent to neighbourhood
+	stateSynced chunk.State // proof is received; chunk removed from sync db; chunk is available everywhere
 }
 
 // NewPss inits the Pss struct with the localstore
 func NewPss(localStore chunk.Store) *Pss {
 	return &Pss{
 		localStore: localStore,
+		tag:        chunk.NewTag(1, "pss-chunks-tag", 0, false),
```

Comment:
pss is meant to be used for multiple clients doing concurrent API calls

Reply:
removed, I created the tag inside the Send function.
Is this correct or should it be done outside and passed into Send? @zelig